docs/html/development/contributing.rst

```diff
@@ -108,11 +108,13 @@ Choosing the type of NEWS entry
 A trivial change is anything that does not warrant an entry in the news file.
 Some examples are: Code refactors that don't change anything as far as the
 public is concerned, typo fixes, white space modification, etc. To mark a PR
-as trivial a contributor simply needs to add a randomly named, empty file to
-the ``news/`` directory with the extension of ``.trivial.rst``. If you are on a
-POSIX like operating system, one can be added by running
-``touch news/$(uuidgen).trivial.rst``. On Windows, the same result can be
-achieved in Powershell using ``New-Item "news/$([guid]::NewGuid()).trivial.rst"``.
+as trivial a contributor simply needs to add a randomly named, non-empty file to
+the ``news/`` directory with the extension of ``.trivial.rst``. Due to a bug in
+the validation, the content of ``.trivial.rst`` must not be empty. Entering a
+single character and a newline in the ``.trivial.rst`` file is sufficient. If
+you are on a POSIX like operating system, one can be added by running
+``touch news/$(uuidgen).trivial.rst``. On Windows, the same result can be achieved
+in Powershell using ``New-Item "news/$([guid]::NewGuid()).trivial.rst"``.
```

Comment:
Shouldn’t the `touch` and `New-Item` suggestions be updated as well since they generate an empty file? It’s probably good anyway to use

```bash
echo "Short message on the change" > news/$(uuidgen).trivial.rst
```

and

```powershell
New-Item -Path "news/$([guid]::NewGuid()).trivial.rst" -Value "Short message on the change"
```

Reply:
Yes, and before someone brings it up, I think it'd be OK to nudge folks to just write a short line about the change. They're gonna do that for the PR title or description anyway.

Reply:
I'll update this in the file

Reply:
I've ran the commands and it works if we add `` `n`` at the end of the message for the Windows suggestion. I've already added this in this ``contributing.rst`` file as well

Reply:
>  if we add `` `n `` at the end of the message for the Windows suggestion.

Ah, right, the final newline… Thanks for checking. Picky bot.